### ISSUE TITLE
[BUG] Export of dataset to pandas loses information of several ratings

### ISSUE THREAD

**Issue by ch-au** [recognai/rubrix#3274]
**Describe the bug**
i have invited several reviewers to one dataset. Apparently everyone will have different "pending" status of a dataset so I get several ratings per record (which is great). However, after translating data to a panda, I have only one review per record

**To Reproduce**
```python
evaluation_data = rg.FeedbackDataset.from_argilla(name="ps123", workspace="myspace")
hf_dataset = evaluation_data.format_as("datasets")
mp = hf_dataset.to_pandas()
```
-> records where i have a list of responses only show one response

**Expected behavior**
either create a row per response or include a list of responses in pandas

**Comment by ch-au** [comment]
@alvarobartt

**Comment by alvarobartt** [comment]
Hi @ch-au thanks for reporting! This should be fixed in the upcoming release of Argilla 👍🏻 Has will be fixed upon #3539 merge! 🎉 Now you should see all the responses for any given question!
